core/src/OpenMPTarget/Kokkos_OpenMPTarget_Exec.hpp

```diff
@@ -1616,6 +1637,12 @@ KOKKOS_INLINE_FUNCTION void parallel_scan(
     // broadcast last value to rest of the team
     member.team_broadcast(accum, team_size - 1);
   }
+#endif
+
+  // for (iType i = 0; i < nchunk; ++i) {
+  // if(omp_get_thread_num() == 0)
+  // lambda(i, accum, true);
+  //}
```

Comment:
Does this need to be removed or in the `#else`?

Reply:
Thats needs to be removed. Done.